dist/yaml/ovnkube-master.yaml

```diff
@@ -59,7 +66,9 @@ spec:
         - mountPath: /var/run/dbus/
           name: host-var-run-dbus
           readOnly: true
-        - mountPath: /var/lib/openvswitch/
+        # ovn db is stored in the pod in /etc/openvswitch
+        # and on the host in /var/lib/openvswitch/
+        - mountPath: /etc/openvswitch/
```

Comment:
This means that `/var/lib/openvswitch` in the container is not persisted. Can you check that nothing is stored there?

Reply:
@squeed The directory was empty when I looked and the only files in it are the ovndb files. /var/lib/openvswitch was not on the host since ovs/ovn is not installed there. I think we are safe. I only mount this on master, the mount on the nodes has been removed.